Goerlinator.md

```diff
@@ -0,0 +1,23 @@
+
+_The Görli Testnet Initiative ("Görli"), in cooperation with the Department of Decentralization ("DoD"), is looking for a ..._ :tada:
+
+# Testnet Coordinator (f/m/*), 
+
+_Or, as we call it: a "Görlinator" (f/m/*)_ :stuck_out_tongue_winking_eye: 
+
+_About_ -- The Görli Testnet Initiative strives to develop, maintain, and sustain networks and tools for application developers in the broader Ethereum ecosystem. To date, the initiative succeeded in bringing the Clique proof-of-authority engine to various core Ethereum clients, to propose and [launch the Görli](https://twitter.com/ethberlin/status/1090991452954521600) and the [Kotti testnets](https://github.com/goerli/testnet#meta-data-kotti-classic), as well as to provide additional tooling.
+
+_Task_ -- The community around the deployed Görli and Kotti testnets often experiences both technical and also organizational issues due to a lack of coordination and the decentralized nature of the projects. We are seeking a person that can take up the tasks of ...
+
+- ... documenting potential issues in the deployed testnets 
+- ... championing mitigation proposals for the clique protocol
+- ... maintaining a _red phone_ with all active validators
+- ... assisting core developers and protocol engineers 
+
+_Meta_ -- This is a part-time role funded by Görli and the DoD and requires high community engagement as well as rudimentary understanding of proof-of-authority networks. You will be rewarded with the most flexible, imaginable working conditions both in time and all three space dimentions.
+
+_Act!_ -- If this is something you were born for, and you can start immediatly, please reach out to [goerli@soc1.cz](mailto:goerli@soc1.cz) with a short letter of intend, a summary of recent experience in this field, earliest possible starting date, and target salary.
```

Comment:
tyop? `letter of intend` -> `letter of intent`. 
or you might want to say `cover letter`